docs/Features.md

```diff
@@ -3,57 +3,13 @@ title: Features
 layout: docs
 ---
 
-Below is a list of features implemented since the [original Theos](https://github.com/theos/theos/tree/legacy) by Dustin Howett. Said features are sorted in ascending order by their implementation date (where the latest additions are at the bottom).
+Below is a list of undocumented features implemented since the [original Theos](https://github.com/theos/theos/tree/legacy) by Dustin Howett. Said features are sorted in ascending order by their implementation date (where the latest additions are at the bottom).
 
 *(To do: These features should be properly documented and this file removed.)*
 
-* Fallback/last resort headers can be placed at `include/_fallback`; this can be used to provide drop-in replacements for missing SDK headers. (rpetrich)
```

Comment:
Maybe this needs to be added to https://github.com/theos/theos.dev/blob/master/docs/Structure.md?

Reply:
Good call!